modules/manage/pages/schema-reg/schema-reg-overview.adoc

```diff
@@ -90,6 +90,31 @@ message Product {
 
 You would want the compatibility setting to accommodate adding new fields without breakage. Adding an optional new field to a schema is inherently backward-compatible. New consumers can process events written with the new schema, and older consumers can ignore it. 
 
+== JSON schema 
+
+All CRUD operations are supported for the JSON schema (`json-schema`), and Redpanda supports https://json-schema.org/specification[all published JSON Schema specifications^], which include:
+
+* draft-04
+* draft-06
+* draft-07
+* 2019-09
+* 2020-12
+
+=== Limitations
+
+Schemas are held in subjects. Subjects have a compatibility configuration associated with them, either directly specified by a user, or inherited by the default. See `PUT /config` and `PUT/config/{subject}` in the xref:api:ROOT:pandaproxy-schema-registry.adoc[Schema Registry API].
+
+If you have inserted a second schema into a subject where the compatibility level is anything but `NONE`, then JSON schemas containing any of the following items are rejected:
+
+* `$ref`
+* `$defs` (`definitions` prior to draft 2019-09)
+* `dependentSchemas` / `dependentRequired` (`dependencies` prior to draft 2019-09)
+* `prefixItems`
+
+Consequently, you cannot https://json-schema.org/understanding-json-schema/structuring[structure a complex schema^].
```

Comment:
@Feediver1  I would add 'using these features' at the end of the sentence since technically you can create a complex schema it just has to be all in one without the benefit of dependencies, references,  definitions and prefixes.